maps/away/ships/heph/cyclops/cyclops_mining_ship.dm

```diff
@@ -0,0 +1,103 @@
+/datum/map_template/ruin/away_site/cyclops_mining_vessel
+	name = "Cyclops Mining Vessel"
+	description = "This bulky vessel is designed and operated by Hephaestus Industries. From asteroid cracking to planetary operations, this ship can do it all. "
+	suffixes = list("ships/heph/cyclops/cyclops.dmm")
+	sectors = list(SECTOR_TAU_CETI, SECTOR_ROMANOVICH, SECTOR_CORP_ZONE, SECTOR_VALLEY_HALE, SECTOR_BADLANDS, SECTOR_SRANDMARR, SECTOR_WEEPING_STARS)
+	spawn_weight = 1
+	ship_cost = 1
+	id = "Cyclops Mining Vessel"
+	shuttles_to_initialise = list(/datum/shuttle/autodock/overmap/cyclops_shuttle)
+	template_flags = TEMPLATE_FLAG_SPAWN_GUARANTEED
+
+/singleton/submap_archetype/cyclops_mining
+	map = "Cyclops Mining Vessel"
+	descriptor = "This bulky vessel is designed and operated by Hephaestus Industries. From asteroid cracking to planetary operations, this ship can do it all. "
+
+// Ship Stuff
+/obj/effect/overmap/visitable/ship/cyclops_mining
+	name = "Cyclops Mining Vessel"
+	class = "HCV"
+	desc = "This bulky vessel is designed and operated by Hephaestus Industries. From asteroid cracking to planetary operations, this ship can do it all. "
+	icon_state = "tramp"
+	moving_state = "tramp_moving"
+	colors = list("#BAB86C", "#8B4000")
+	max_speed = 1/(2 SECONDS)
+	burn_delay = 1 SECONDS
+	vessel_mass = 5000
+	fore_dir = SOUTH
+	vessel_size = SHIP_SIZE_SMALL
+	initial_restricted_waypoints = list(
+        "Cyclops Shuttle" = list("nav_hangar_cyclops")
```

Comment:
Please add RP fluff fields like `designer`, `weapons`, `shiptype`, as described in `code\modules\overmap\overmap_object.dm`